bin/ch-fromhost

```diff
@@ -190,6 +239,14 @@ while [ $# -gt 0 ]; do
             queue_files "$out"
             shift
             ;;
+        --host-ofi)
+            if [ -z "$CH_FROMHOST_OFI" ]; then
+                hint_1="hint: use --ofi to target specific -fi.so provider; or"
+                hint_2="hint: see FAQ: site OFI providers (FIXME:)"
+                fatal "CH_FROMHOST_OFI not set\n%s\%s" "$hint_1" "$hint_2"
```

Comment:
Do you mean? `\n%s\n%s`? Regardless, it might be good to try to edit the hint down to a single line.